.eslintrc

```diff
@@ -1,160 +1,153 @@
-      ],
-      "plugins": [
-        "deprecation"
-      ],
-      "rules": {
-        "prettier/prettier": [
-          "error",
-          {
-            "endOfLine": "auto"
-          }
-        ],
-        // OUR RULES
-        "@typescript-eslint/naming-convention": [
-          "error",
-          {
-            "selector": "enumMember",
-            "format": [
-              "PascalCase",
-              "camelCase",
-              "UPPER_CASE"
-            ]
-          }
-        ],
-        // Would be nice to have these three but requires refactoring of old code
-        "@typescript-eslint/no-redundant-type-constituents": "off",
-        "@typescript-eslint/no-base-to-string": "off",
-        "@typescript-eslint/no-unsafe-enum-comparison": "off",
-        "@typescript-eslint/explicit-function-return-type": "off",
-        // on aimerait bien dire oui sauf pour void
-        "@typescript-eslint/explicit-module-boundary-types": "off",
-        // on aimerait bien dire oui sauf pour void
-        "@angular-eslint/no-input-rename": "off",
-        "@typescript-eslint/no-unused-vars": [
-          "error",
-          {
-            "argsIgnorePattern": "^_",
-            "ignoreRestSiblings": true
-          }
-        ],
-        "@typescript-eslint/unbound-method": "off",
-        "@typescript-eslint/type-annotation-spacing": [
-          "error"
-        ],
-        "no-console": [
-          "error",
-          {
-            "allow": [
-              "warn",
-              "error"
-            ]
-          }
-        ],
-        "curly": "error",
-        "deprecation/deprecation": "warn",
-        "no-restricted-imports": [
-          "error",
-          {
-            "paths": [
-              "rxjs/Rx",
-              "@ngneat/spectator"
-            ]
-          }
-        ],
-        "max-classes-per-file": "off",
-        "quote-props": [
-          "error",
-          "as-needed"
-        ],
-        "quotes": [
-          "error",
-          "single",
-          {
-            "avoidEscape": true,
-            "allowTemplateLiterals": true
-          }
-        ],
-        "space-before-function-paren": [
-          "error",
-          {
-            "anonymous": "never",
-            "asyncArrow": "always",
-            "named": "never"
-          }
-        ],
-        // BASIC RULES
-        "@angular-eslint/component-selector": [
-          "error",
-          {
-            "type": "element",
-            "prefix": "lu",
-            "style": "kebab-case"
-          }
-        ],
-        "@angular-eslint/directive-selector": [
-          "error",
-          {
-            "type": "attribute",
-            "prefix": "lu",
-            "style": "camelCase"
-          }
-        ],
-        "@angular-eslint/no-host-metadata-property": "off"
-      }
-    },
-    {
-      "files": [
-        "*.html"
-      ],
-      "extends": [
-        "plugin:@angular-eslint/template/recommended",
-        "plugin:prettier/recommended"
```

Comment:
We use this exact configuration in other projects and it doesn't raise any issue. I don't understand why removing `prettier/recommended` here solves anything 🤔

Reply:
Me neither, I tested on several files and it makes no sense.

If I add it back, the result format is derping, if I revert all the changes and exclude eslint for formatting (and thus format with only prettier), it works fine. Now if I enable prettier only in this eslint config, I get an error due to `<` being unexpected, showing that somehow it's considering that as a ts file???

Reply:
Isn't the problem that 
- in prettier config : `.component.html` and `.page.html` files are used with the angular parser
- in prettier config : `.html` files are used with the html parser
- in eslint config : all `.html` files are used with the same rules ?

Maybe there should be the same split between files in eslint config ? 🤔 and `.html` files would just use `prettier:prettier/recommended` while `.component.html` and `.page.html` files would use both angular-eslint and prettier ?

Or maybe just use `.html` in prettier like in the recommended config by angular eslint : https://github.com/angular-eslint/angular-eslint/blob/main/docs/CONFIGURING_ESLINTRC.md#notes-for-eslint-plugin-prettier-users